### PR TITLE
added examples

### PR DESCRIPTION
**Pull request by vighnesh-tirakapadi**

### CODE REVIEW

File: src/components/PopupModal/ExchangePopup/ExchangePopup.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      {modalbutton && (
+        <DefaultButton
+          type="button"
+          hover
+          variant={ButtonVariant.DefaultPrimary}
+          size={ButtonSize.Small}
+          onClick={toggleMenu}
+          test-id="ToggleBtn"
+        >
+          Open Modal
```
Comment (deeporiontechnology) [inline src/components/PopupModal/ExchangePopup/ExchangePopup.tsx:82]: Can we take this label from props only

File: src/components/PopupModal/ExchangePopup/ExchangePopup.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+              {button01 && (
+                <DefaultButton
+                  hover
+                  size={ButtonSize.Small}
+                  className="ml-2 mr-2 w-1/4"
+                  variant={ButtonVariant.DefaultPrimary}
+                  onClick={toggleMenu}
+                  data-testid="button01"
+                >
+                  button
```
Comment (deeporiontechnology) [inline src/components/PopupModal/ExchangePopup/ExchangePopup.tsx:124]: Same about this

File: src/components/PopupModal/ExchangePopup/ExchangePopup.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+              {button02 && (
+                <DefaultButton
+                  hover
+                  variant={ButtonVariant.DefaultPrimary}
+                  size={ButtonSize.Small}
+                  className="ml-2 mr-2 w-1/4"
+                  onClick={toggleMenu}
+                  data-testid="button02"
+                >
+                  button
```
Comment (deeporiontechnology) [inline src/components/PopupModal/ExchangePopup/ExchangePopup.tsx:136]: same here

File: src/components/PopupModal/ExchangePopup/ExchangePopup.tsx (hunk 2 above)
Comment (vighnesh-tirakapadi) [reply src/components/PopupModal/ExchangePopup/ExchangePopup.tsx]: Done sir

**Pull request merged**